test/integration/project_test.go

```diff
@@ -157,6 +157,10 @@ func TestProjectBranches(t *testing.T) {
 
 func createProject(t *testing.T, tags map[string]string, groups []string) (string, string) {
 	projectName := getProjectNameForTest() + "_for_project"
```

Comment:
create new project - need to assign with the project name - GenerateRandomProjectNameForScan()

Reply:
same response as above - refactor was made in **_getProjectNameForTest_** so now it's working as expected for all tests

Reply:
so now you creating a new project but with an exciting name? it is supposed to fail or only update the project. I think you should use here GenerateRandomProjectNameForScan

Reply:
we shouldn't random a new name here, bacause we don't want to create a newproject
only if we want a new project we use it

Reply:
@AlvoBen - please do the CR for all the tests